src/style.css

```diff
@@ -0,0 +1,181 @@
+html {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  max-width: 198px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.photo {
+  height: 134px;
+  width: 160px;
+}
+
+.card__photo {
+  margin-top: 32px;
+}
+
+.product {
+  margin-top: 40px;
+
+  display: flex;
+  flex-direction: column;
+}
+
+.description {
+  margin: 0;
+}
+
+.product__name {
+  width: 166px;
+  height: 36px;
+
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.product__code {
+  margin-top: 4px;
+
+  font-size: 10px;
+  font-weight: 400;
+
+  color: #616070;
+}
+
+.review {
+  margin-top: 16px;
```

Comment:
no external styles for blocks

Reply:
for pixel-accuracy you need to use 17px
and as Ihor wrote try to apply a mix of styles to avoid setting external styles for blocks
likes of
```
.marginTop17 {
  margin-top: 17px;
}
```
and combine classes on html
`< class="review marginTop17">`